source/operatingsystems/linux/elasticsearch/elasticsearch.md

```diff
@@ -102,6 +102,19 @@ network.host: 10.0.0.17
 
 The Elasticsearch service will need a restart after this change.
 
+### Memory Limit/Heap Size
+On recent versions of Elasticsearch, the default memory allocation is 32GB which can cause issues. We recommend that you set to 2GB in `/etc/elasticsearch/jvm.options`
+
+```bash
+vim /etc/elasticsearch/jvm.options
+
+ -Xms2g
+ -Xmx2g
+
+```
+
+The Elasticsearch service will need a restart after this change.
```

Comment:
Worth including the restart command within this, if referencing it.

Reply:
Changed them now